build/build-docker-deploy.sh

```diff
@@ -24,7 +30,7 @@ if [ "${1-}" = "docker" ]; then
 fi
 
 # Build the cockroach and test binaries.
-$(dirname $0)/builder.sh $0 docker
+$(dirname $0)/builder.sh $(path_for_docker) docker
```

Comment:
They allow the script to be independent of the current directory. If not these changes, the script should check that the current directory is the top-level cockroach directory which is the only place it currently works.